README.md

```diff
@@ -27,6 +27,9 @@ command `CREATE DATABASE chitter_test;`;
 * Connect to the database using the psql command `\c chitter_test`
 * Run the query we have saved in the file 01_create_chitter_table.sql
 
```

Comment:
Good use of the README to explain how to connect to database. Easy to follow instructions.